src/extract.js

```diff
@@ -229,16 +229,17 @@ exports.Extractor = class Extractor {
 
   processStrings(extractedData) {
     for (const d of extractedData) {
-      if (!this.items[d.text]) {
-        this.items[d.text] = {};
+      const msgid = d.text || d.msgid;
```

Comment:
@vperron, I'm not sure that is the best way to fix incorrect processing strings from <script> section. There are 2 cases when the `processStrings` invokes:
- when parsing template using `extractor.parse`
- when parsing <script> section using `extractor.parseVueJavascript`

In both cases we have different data structures. It makes sence to process them separately or transform extracted data from JavaScript to the same structure as after parsing template.

Reply:
I agree. I was just a little too lazy, and did not have time, to correctly refactor this in the time. If you have any kind of good proposition that for instance, reuses some of the internal logic thanks to cleaner data structures, and has two differents calls for each case, I'll take it !